### ISSUE TITLE
“Share App Usage” switch is not programmatically associated with its text label [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#275]
No accessible name is announced upon focusing the “Share App Usage” switch on the App Settings -> Help Improve This App screen. The visible text label and the switch control are two separate tab stops.

![](https://user-images.githubusercontent.com/4437084/92060369-8cea5700-ed61-11ea-8f0e-c552eb2db0d3.png)

Programmatically associate the label by including it in the accessibilityLabel attribute on the switch. Hide the visible text label from VoiceOver users via the accessibilityElementsHidden attribute, as not to announce the label twice to said users.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Share App Usage</Text>
</View>
<Switch accessibilityLabel='Share App Usage' />
```

**Comment by willslone** [comment]
I retested and can confirm the issue has been resolved.
